maps/submaps/surface_submaps/wilderness/wilderness.dm

```diff
@@ -151,4 +152,9 @@
 	name = "Manhack Rock"
 	desc = "A rock filled with nasty Synthetics."
 	mappath = 'maps/submaps/surface_submaps/wilderness/MHR.dmm'
-	cost = 15
\ No newline at end of file
+	cost = 15
+
+/datum/map_template/surface/GovPatrol
+	name = "GovPatrol"
+	desc = "A long lost SifGuard ground survey patrol. Now they have you guys!"
+	mappath = 'maps/submaps/surface_submaps/GovPatrol.dmm'
```

Comment:
Think this needs to be ``'maps/submaps/surface_submaps/wilderness/GovPatrol.dmm'`` or the game won't find it.